src/SourceCode.Clay.Text/StringBuilderExtensions.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Text;
+
+namespace SourceCode.Clay.Text
+{
+    public static class StringBuilderExtensions
+    {
+        public static StringBuilder AppendFormatLine(this StringBuilder sb, string format, object arg0)
+            => sb?.AppendFormat(format, arg0).AppendLine();
+
+        public static StringBuilder AppendFormatLine(this StringBuilder sb, string format, object arg0, object arg1)
+            => sb?.AppendFormat(format, arg0, arg1).AppendLine();
+
+        public static StringBuilder AppendFormatLine(this StringBuilder sb, string format, object arg0, object arg1, object arg3)
+            => sb?.AppendFormat(format, arg0, arg1, arg3).AppendLine();
+
+        public static StringBuilder AppendFormatLine(this StringBuilder sb, string format, params object[] args)
+            => sb?.AppendFormat(format, args).AppendLine();
+
+        public static StringBuilder AppendFormatLine(this StringBuilder sb, IFormatProvider provider, string format, object arg0)
+            => sb?.AppendFormat(provider, format, arg0).AppendLine();
+
+        public static StringBuilder AppendFormatLine(this StringBuilder sb, IFormatProvider provider, string format, object arg0, object arg1)
+            => sb?.AppendFormat(provider, format, arg0, arg1).AppendLine();
+
+        public static StringBuilder AppendFormatLine(this StringBuilder sb, IFormatProvider provider, string format, object arg0, object arg1, object arg3)
+            => sb?.AppendFormat(provider, format, arg0, arg1, arg3).AppendLine();
+
+        public static StringBuilder AppendFormatLine(this StringBuilder sb, IFormatProvider provider, string format, params object[] args)
+            => sb?.AppendFormat(provider, format, args).AppendLine();
+
+        public static StringBuilder AppendBuilder(this StringBuilder sb, StringBuilder arg)
+        {
+            if (sb == null || sb.Length == 0) return arg;
```

Comment:
This is odd to me. I would expect the main reason to pass a StringBuilder would be for it to be modified. At the very lease, I wouldn't be surprised if it was.